external/fv3fit/fv3fit/keras/_models/recurrent.py

```diff
@@ -430,12 +433,12 @@ def get_target_state(self, X):
     @property
     def predictor_model(self) -> "StepwiseModel":
         predictor_model = StepwiseModel(
-            self.sample_dim_name,
             tuple(self.input_packer.pack_names)
             + tuple(self.prognostic_packer.pack_names),
             self.output_variables,
             self._stepwise_output_metadata,
             self.predict_keras_model,
+            self.sample_dim_name,
```

Comment:
suggestion: if you update these to use keyword arguments for the variables (just here, you don't need to change the call signature) we can bugs from mis-ordered arguments.